sql/src/main/java/org/apache/druid/sql/calcite/planner/ExplainAttributes.java

```diff
@@ -107,6 +112,7 @@ public String getClusteredBy()
    */
   @Nullable
   @JsonProperty
+  @JsonInclude(JsonInclude.Include.NON_NULL)
```

Comment:
Would it be work to have this annotation just once at the class level instead?

Reply:
Yeah, it seems like we do a mix of method-level and class-level annotations in the codebase. Since we already do `@Nullable` consistently for all the methods in this class, I just stuck with the former.